src/patterns/InPlaceFilters/FilterButton.tsx

```diff
@@ -0,0 +1,204 @@
+import * as Popover from '@radix-ui/react-popover';
+import classnames from 'classnames';
+import React from 'react';
+import Button from '../../components/Button/Button';
+import { ButtonVariants } from '../../components/Button/types';
+import Drawer from '../../components/Drawer/Drawer';
+import Filter from '../../components/Filter/Filter';
+import FilterHeader from '../../components/Filter/FilterHeader';
+import FilterInput from '../../components/Filter/FilterInput';
+import { SSRMediaQuery } from '../../providers/SeldonProvider/utils';
+import { px } from '../../utils';
+import FilterMenu from '../FilterMenu/FilterMenu';
+import { FilterButtonDisplay } from './FilterButtonDisplay';
+import { FilterDropdown } from './FilterDropdown';
+import { FilterButtonProps, FilterDimension, FilterType } from './types';
+import { countActiveFilters } from './utils';
+
+export const FilterButton = React.forwardRef<HTMLButtonElement, FilterButtonProps>(
```

Comment:
This seems like more than just a button, since it contains a drawer as well so I think `<FilterButtonDisplay` should probably be renamed `<FilterButton` and we need to figure out a good name that represents this